src/pages/ResetPassword/index.tsx

```
@@ -2,101 +2,160 @@ import type React from "react"
 
-    try {
-      const response = await axios.post("http://localhost:3000/auth/reset-password", {
-        token,
-        password: newPassword,
-      })
-      setSuccess("Password changed successfully. Redirecting...")
-      setTimeout(() => navigate("/login"), 2500)
-    } catch (err: any) {
-      setError(err?.response?.data?.message || "An error occurred.")
+        try {
+            const response = await axios.post("http://localhost:3000/auth/reset-password", {
```

ESLint

[error] 'response' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)